packages/manager/apps/web/client/app/domain/list/style.scss

```diff
@@ -0,0 +1,9 @@
+.list-domains-datagrid {
+  .domains-nameserver-column {
+    width: 15vw;
+  }
```

Comment:
Cannot be managed with bootstrap css ?

Reply:
I think the bootstrap sizing is relative to the parent but we need to be relative to screen size